examples/src/bin/ble_bas_peripheral.rs

```diff
@@ -28,13 +31,18 @@ async fn softdevice_task(sd: &'static Softdevice) {
 struct BatteryService {
     #[characteristic(uuid = "2a19", read, write, notify)]
     battery_level: u8,
-    #[characteristic(uuid = "3a4a1f7e-22d8-11eb-a3aa-1b3b1d4e4a0d", read, write, notify)]
+}
+
+#[nrf_softdevice::gatt_server(uuid = "9e7312e0-2354-11eb-9f10-fbc30a62cf38")]
```

Comment:
ah but now see its weird. You call `gatt_server` twice. Should probably be called `gatt_service`?